### PR TITLE
Implement DELETE request to cucumber tests

### PR DESCRIPTION
**Pull request by JessickaMoya**

### CODE REVIEW

File: src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java [hunk 1]
```diff
@@ -79,10 +106,17 @@ public void theResponseStatusCodeShouldBeOnPatchAccountRequest(final String stat
 
     @When("I execute the delete Account request on {string} endpoint")
     public void iExecuteTheDeleteAccountRequestOnEndpoint(final String endpoint) {
+        requestBuilder
```
Comment (Raymundogs) [inline src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java:109]: parameter not used. It should be used or deleted.

File: src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java [hunk 2]
```diff
@@ -79,10 +106,17 @@ public void theResponseStatusCodeShouldBeOnPatchAccountRequest(final String stat
...
+                .addEndpoint("/Account/{accountID}")
+                .addPathParams("accountID", responseObject.getId().toString())
+                .addMethod(ApiMethod.DELETE)
+                .build();
+        apiResponse = ApiManager.execute(requestBuilder.build());
     }
 
     @Then("The response status code should be {string} on delete Account request")
     public void theResponseStatusCodeShouldBeOnDeleteAccountRequest(final String statusCode) {
+        Assert.assertEquals(apiResponse.getStatusCode(), 204);
```
Comment (Raymundogs) [inline src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java:119]: parameter not used. It should be used or deleted.

File: src/test/java/rest/salesforce/org/steps/ContractsApiSteps.java [hunk 3]
```diff
@@ -59,6 +117,9 @@ public void theResponseStatusCodeShouldBeOnGetAllContractsRequest(final String s
 
     @Given("I build a {string} request for a Contract")
     public void iBuildARequestForAContract(final String apiMethod) {
+        requestBuilder
```
Comment (Raymundogs) [inline src/test/java/rest/salesforce/org/steps/ContractsApiSteps.java:120]: api method is not set on this step

File: src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java (hunk 1 above)
Comment (JessickaMoya) [reply src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java:109]: Thanks, I have just fixed it

File: src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java (hunk 2 above)
Comment (JessickaMoya) [reply src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java:119]: Thanks, in the next PR I will fix it

File: src/test/java/rest/salesforce/org/steps/ContractsApiSteps.java (hunk 3 above)
Comment (JessickaMoya) [reply src/test/java/rest/salesforce/org/steps/ContractsApiSteps.java:120]: Thanks, in the next PR I will fix it

**Review by cristianfchq — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

**Pull request merged**